hvplot/__init__.py

```diff
@@ -209,6 +209,54 @@ def _hook_patch_docstrings(backend):
 
 @_wraps(_pn.bind)
 def bind(function, *args, **kwargs):
+    """
+    Returns a *reactive* function that *depends* on widgets, parameters or Python objects.
+    This means that the "reactive" function can be automatically invoked whenever the underlying
+    widget or parameter values change. The widgets can be Panel or IpyWidgets.
+    
+    `hv.bind` can be **used to start your `.interactive` pipeline** by loading data depending
+    on widgets or pameters values.
+    
+    Reference: https://github.com/holoviz/hvplot/issues/825
+
+    :Example:
+
+    >>> import pandas as pd
+    >>> import panel as pn
+
+    >>> import hvplot
+
+    >>> pn.extension()
```

Comment:
Is there a reason for the spaces between these lines?

Reply:
I run the example to test it works. I always use `black` to autoformat. I did not think about that this creates spaces in the examples when I copy-paste them back.

I will remove the space.

Reply:
After having looked at the code I restructured it a bit. I tried imagining being a user studying the example or me trying to to explain the example to a colleague. And for that I would like a bit of space to separate imports and distinct sections of the code.

Hope you like it. Otherwise let me know.